css/styles.css

```diff
@@ -77,11 +121,162 @@ button:active {
   line-height: 21px;
 }
 
-.footerDeco {
+.middleDeco {
   background-image: url(../resources/image_geometry_2.svg);
   background-repeat: no-repeat;
   background-size: cover;
   width: 100%;
   height: 0;
   padding-top: 50%;
 }
+
+.greenul {
+  display: flex;
+  justify-content: flex-start;
+  list-style: none;
+  padding-left: 27px;
+}
+
+.greenli {
+  font-size: 14px;
+  color: #1a2236;
+  border: 1px solid #98dede;
+  background-color: #98dede;
+  margin-right: 3px;
+}
+
+.fullWidth {
+  margin-top: -2px;
+  margin-bottom: -6px;
+  width: 100%;
+}
+
+.blueBox {
+  background: #1a2236;
+  display: grid;
+  justify-items: flex-start;
+  width: 100%;
+}
+
+.projectName {
+  font-family: PoppinsLight, sans-serif;
+  font-size: 20px;
+  color: white;
+  padding-left: 25px;
+  margin-bottom: 4px;
+  background-color: #1a2236;
+}
```

Comment:
Let's remember to add more spacing to the design to match the required Figma template 👍 

|  Your design   |  Figma design   |
| -------------- | -------------- |
| <img width="537" alt="Screenshot 2022-10-31 at 18 38 13" src="https://user-images.githubusercontent.com/66112760/199072994-1e147d7e-0af7-4791-a294-82b36c79d211.png">   | <img width="459" alt="Screenshot 2022-10-31 at 18 38 52" src="https://user-images.githubusercontent.com/66112760/199073106-79a10034-8967-4216-a205-0094b03b33ad.png">   |